numpy/lib/tests/test_function_base.py

```diff
@@ -1756,7 +1756,23 @@ def test_simple(self):
         assert_array_equal(unwrap([1, 1 + 2 * np.pi]), [1, 1])
         # check that unwrap maintains continuity
         assert_(np.all(diff(unwrap(rand(10) * 100)) < np.pi))
-
+    
+    def test_period(self):
+        # check that unwrap removes jumps greater that 255
+        assert_array_equal(unwrap([1, 1 + 256], interval_size=255), [1, 2])
+        # check that unwrap maintains continuity
+        assert_(np.all(diff(unwrap(rand(10) * 1000, interval_size=255)) < 255))
+        # check simple case
+        simple_seq = np.array([0, 75, 150, 225, 300])
+        wrap_seq = np.mod(simple_seq, 255)
+        assert_array_equal(unwrap(wrap_seq, interval_size=255), simple_seq)
+        # check custom discont value
+        uneven_seq = np.array([0, 75, 150, 225, 300, 430])
+        wrap_uneven = np.mod(uneven_seq, 250)
+        no_discont = unwrap(wrap_uneven, interval_size=250)
+        assert_array_equal(no_discont, [0, 75, 150, 225, 300, 180])
+        sm_discont = unwrap(wrap_uneven, interval_size=250, discont=140)
+        assert_array_equal(sm_discont, [0, 75, 150, 225, 300, 430])
```

Comment:
Can you add an explicit test of `dtype` in here too?

Reply:
Like `assert(sm_discont.dtype == wrap_uneven.dtype)`? Or is there a better assertion function to do that?

Reply:
Now that we use pytest, `assert sm_discont.dtype == wrap_uneven.dtype` would be fine.